src/python/pants/engine/legacy/address_mapper.py

```diff
@@ -33,17 +31,15 @@ def __init__(self, scheduler, engine, build_root):
     self._build_root = build_root
 
   def scan_build_files(self, base_path):
-    subject = DescendantAddresses(base_path)
-    selector = SelectDependencies(BuildFiles, BuildDirs, field_types=(Dir,))
-    request = self._scheduler.selection_request([(selector, subject)])
+    request = self._scheduler.execution_request([BuildFilesCollection], [(DescendantAddresses(base_path))])
```

Comment:
I think this method is now unused inside pants itself. It's not marked public. Could I just remove it rather than adjusting to it's use case?

I think it is also untested, which makes me 🙁